lisa/sut_orchestrator/libvirt/transformers.py

```diff
@@ -571,7 +570,11 @@ def _install_libvirt(runbook: schema.TypedSchema, node: Node, log: Logger) -> No
         libvirt_version = libvirt_installer._get_version()
         log.info(f"Already installed! libvirt version: {libvirt_version}")
         _fix_mariner_installation(node=node)
-        node.reboot(time_out=900)
+
+    node.execute("systemctl enable libvirtd", shell=True, sudo=True)
+    node.execute("systemctl enable virtnetworkd", shell=True, sudo=True)
+    log.info("Enabling libvirtd and virtnetworkd services")
```

Comment:
The "Enabling ..." should be above of the node.execute lines. After the node.execute lines, it should be "Enabled ...".

Reply:
done!